adwords/report-builder.js

```diff
@@ -48,7 +48,7 @@ class AdwordsReportBuilder {
      * @param fields {array}
      */
     buildFields(selector, fields) {
-        for (var index in fields) {
+        for (var index = 0; index < fields.length; index++) {
```

Comment:
There are times where fields / filters / filter values are null. using .length without checking if they exist will end up causing code to crash. Can you switch to maybe a lodash `each` or use `(fields || []).forEach`.